packages/cozy-client/src/store/queries.js

```diff
@@ -171,11 +176,14 @@ const queries = (state = {}, action, documents = {}) => {
     return mapValues(state, queryState => {
       if (queryState.id == action.queryId) {
         return query(queryState, action)
-      } else {
+      } else if (documentsChanged) {
         return updater(queryState)
+      } else {
+        return queryState
       }
     })
-  } else if (isReceivingMutationResult(action) || isReceivingData(action)) {
+  }
+  if (isReceivingMutationResult(action)) {
```

Comment:
We can't `receivingData` anymore?

Reply:
We couldn't before either. `isReceivingData` is more an action with type `RECEIVE_QUERY_RESULT`, and those get caught by `isQueryAction` just above.